launcher/Application.cpp

```diff
@@ -410,6 +413,47 @@ Application::Application(int &argc, char **argv) : QApplication(argc, argv)
                 " " "|" " "
                 "%{if-category}[%{category}]: %{endif}"
                 "%{message}");
+      
+        bool foundLoggingRules = false;
+        
+        auto logRulesFile = QStringLiteral("qtlogging.ini");
+        auto logRulesPath = FS::PathCombine(dataPath, logRulesFile);
+        
+        qDebug() << "Testing" << logRulesPath << "...";              
+        foundLoggingRules = QFile::exists(logRulesPath);
+
+        // search the dataPath()
+        
+        if(!foundLoggingRules && !isPortable() && dirParam.isEmpty()) {
+            logRulesPath = QStandardPaths::locate(QStandardPaths::AppDataLocation, logRulesFile);
```

Comment:
```suggestion
            logRulesPath = QStandardPaths::locate(QStandardPaths::AppDataLocation, "../" + logRulesFile);
```

Note that `QStandardPaths::AppDataLocation` will include the Qt organization name in the path (`PrismLauncher/PrismLauncher`), so we need to go up a level. Though I am not sure if this works as expected.

Reply:
it certainly appears to (I've tested it locally)